client/ayon_core/hosts/maya/plugins/create/create_animation_pointcache.py

```diff
@@ -0,0 +1,156 @@
+from maya import cmds
+
+from ayon_core.hosts.maya.api import lib, plugin
+
+from ayon_core.lib import (
+    BoolDef,
+    NumberDef,
+)
+from ayon_core.pipeline import CreatedInstance
+
+
+def _get_animation_attr_defs(cls):
+    """Get Animation generic definitions."""
+    defs = lib.collect_animation_defs()
+    defs.extend(
+        [
+            BoolDef("farm", label="Submit to Farm"),
+            NumberDef("priority", label="Farm job Priority", default=50),
+            BoolDef("refresh", label="Refresh viewport during export"),
+            BoolDef(
+                "includeParentHierarchy",
+                label="Include Parent Hierarchy",
+                tooltip=(
+                    "Whether to include parent hierarchy of nodes in the "
+                    "publish instance."
+                )
+            ),
+            BoolDef(
+                "includeUserDefinedAttributes",
+                label="Include User Defined Attributes"
+            ),
+        ]
+    )
+
+    return defs
+
+
+def extract_alembic_attributes(node_data, class_name):
+    """This is a legacy transfer of creator attributes to publish attributes
+    for ExtractAlembic/ExtractAnimation plugin.
```

Comment:
To me this function name does not describe what it does. e.g. `convert_legacy_alembic_creator_attributes` would be better maybe?

Reply:
done